docs/how-tos/trigger-sync.md

```diff
@@ -1,61 +1,93 @@
 ---
-title: "Trigger Cloud Sync Conditionally"
-linkTitle: "Trigger Sync"
-description: "Trigger cloud sync to sync captured data conditionally."
-weight: 20
+title: "How to trigger cloud sync conditionally"
+linkTitle: "Trigger data sync conditionally"
+description: "Trigger cloud sync to sync captured data when custom conditions are met."
+weight: 44
 type: "docs"
 tags: ["data management", "cloud", "sync"]
+images: ["/services/icons/data-cloud-sync.svg"]
 aliases:
   - /data/trigger-sync/
+  - /services/data/trigger-sync/
 no_service: true
-# SME: Alexa Greenberg
 ---
 
-You can use a {{< glossary_tooltip term_id="module" text="module" >}} to sync data only when a certain logic condition is met, instead of at a regular time interval.
+You may want to sync data only when a certain logic condition is met, instead of at a regular time interval.
 For example, if you rely on mobile data but have intermittent WiFi connection in certain locations or at certain times of the day, you may want to trigger sync to only occur when these conditions are met.
 Or, you may want to trigger sync only when your machine detects an object of a certain color.
 The code for both of these examples is provided by the [trigger-sync-examples module](https://github.com/viam-labs/trigger-sync-examples-v2), and you can [create your own module](/how-tos/create-module/) if you want to use different logic.
 
+To set up conditional syncing you need to:
+
+1. Create or use an existing module that implements a `sensor` component that uses your custom logic to determine when to enable and disable sync.
+   While this sensor is not sensing the time it _senses_ whether the data manager should sync or not.
+2. Change the configuration of the data manager to enable selective sync.
+
 {{% alert title="Note: How sync is triggered" color="note" %}}
 
 Regardless of the specifics of your trigger sync logic, to trigger sync you need to pass `true` to the [CreateShouldSyncReading function](https://pkg.go.dev/go.viam.com/rdk/services/datamanager#CreateShouldSyncReading) within the definition of your modular sensor's `Readings` function.
 See examples in the `Readings` function of the [time-interval-trigger code](https://github.com/viam-labs/trigger-sync-examples-v2/blob/main/time-interval-trigger/selective_sync/selective_sync.go) and the [color-trigger code](https://github.com/viam-labs/trigger-sync-examples-v2/blob/main/color-trigger/selective_sync/selective_sync.go).
 
 {{% /alert %}}
 
+This page covers how to use the `sync-at-time` module.
+You can use this as an example if you use or create a similar module.
+
+{{% alert title="In this page" color="tip" %}}
+
+1. [Add the `sync-at-time` "sensor"](#add-sensor-to-determine-when-to-sync)
+2. [Configure the data manager to sync based on the sensor](#configure-the-data-manager-to-sync-based-on-sensor)
+3. [Test your sync configuration](#test-your-sync-configuration)
+
+{{% /alert %}}
+
 ## Example: `sync-at-time`
```

Comment:
I'd really like to keep the basic structure of all the how tos the same. So Prerequisites etc should be H2s. Why don't  we just remove this heading.

```suggestion
```

Reply:
The reason I did this is to make this how-to just a bit more generalizable, and make it clear that this is one module but you could do the same thing with your own. I think this heading helps that in case people are skimming and don’t read that part of the text

Reply:
Ok keep it. The others need to become h2s though. There is no reason why they need to be h3s

Reply:
Ah okay. I was thinking of them as being part of the example but ok